lib/galaxy/config/sample/tool_conf.xml.sample

```diff
@@ -136,7 +136,7 @@
     <tool file="interactive/interactivetool_cellxgene.xml" />
     <tool file="interactive/interactivetool_ethercalc.xml" />
     <tool file="interactive/interactivetool_hicbrowser.xml" />
-    <tool file="interactive/interactivetool_jupyter_notebook_0.3.xml" />
+    <tool file="interactive/interactivetool_jupyter_notebook_1.x.x.xml" />
```

Comment:
Can you put this at a fixed version please ?

Reply:
That's fine, but please call it `interactive/interactivetool_jupyter_notebook_1.0.0.xml`, we'll have to ship older versions as well.

Reply:
I think its a great idea to ship older version, but we haven't been doing that at all for any interactive tools.